content/millennium/r4/scheduling/appointment.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Appointment | R4 API
+---
+
+# Appointment
+
+* TOC
+{:toc}
+
+## Overview
+
+The Appointment resource provides the ability to retrieve information about appointments, write new appointments, and update the status of existing appointments.
+
+Date is required when searching by patient, practitioner, or location.
+
+When integrating your application with a client’s production environment you will work with the client to determine the Practitioner and Location ids (Millennium personnel and location codes, respectively) which they want to make available to third-party applications for enabling scheduling functionality.
+
+When updating an appointment, the resource provides the ability to change the Appointment.status from booked to arrived, checked-in, or cancelled.
+
+The status of arrived does not change the state of the millennium appointment but instead updates the patient tracking status to a value of Arrived.
+
+R4 exposes a native checked-in status which allows for an appointment to transition from the booked state to the checked-in state. This is a change from the R4 value mapping.
+
+When updating an appointment, the resource provides the ability to change the Appointment.status in the following sequences:
+
+ - From Booked to Arrived, Checked-In, or Cancelled
```

Comment:
use no space and an asterisk for these bullets

Reply:
changed, here: https://github.com/cerner/fhir.cerner.com/pull/257/commits/1b4f8ec57aee0a033fdbaeabdc7125f26d9a97c7#diff-1d7df8ceb9f0ae8471323516e037c662R26